venues/NIPS.cc/2017/Workshop/MLITS/params.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
same here. what's the difference between these two json files?

Reply:
One says 
"MLITS Workshop" - "Machine Learning for Intelligent Transportation Systems", and is due on Oct 12 the other says 
"Autodiff Workshop" - "The future of gradient-based machine learning software and techniques" and is due on Oct 14.

Reply:
Yes, I understand that there are two different conferences, but there are four total files, two of which seem redundant:

venues/NIPS.cc/2017/Workshop/MLITS/params.json seems redundant with venues/NIPS.cc/2017/Workshop/MLITS/data/NIPS2017_MLITS.json,

and 

venues/NIPS.cc/2017/Workshop/Autodiff/params.json seems redundant with
venues/NIPS.cc/2017/Workshop/Autodiff/data/NIPS2017_autodiff.json